__tests__/buildTranslationFiles.spec.ts

```diff
@@ -197,10 +218,11 @@ describe('buildTranslationFiles', () => {
       it('should work with service', () => {
         let expected = gKeys(19);
         expected['20.21.22.23'] = 'missing';
-        expected['24'] = 'missing';
-        expected['25'] = 'missing';
+        ['24', '25', '26', '27', '28', '29', '30', '31', '32', '33'].forEach(
```

Comment:
```suggestion
        for(let i = 24; i < 34; i++) {
          expected[i] = m;
        }  
```

Reply:
since we seem to be using numbers as keys frequently, I just modified our gKeys to accept a start/end range.